src/checkqueue.h

```diff
@@ -81,7 +81,7 @@ class CCheckQueue
                     fAllOk &= fOk;
                     nTodo -= nNow;
                     if (nTodo == 0 && !fMaster)
-                        // We processed the last element; inform the master he can exit and return the result
+                        // We processed the last element; inform the master he or she can exit and return the result
```

Comment:
@Diapolo It was prematurely ack'd. See the long discussion here: https://github.com/bitcoin/bitcoin/pull/5730 . It will be fixed.